### PR TITLE
Added Synth recipes 

### PR DESCRIPTION
**Pull request by R3P0FFXI**

### CODE REVIEW

File: sql/synth_recipes.sql [hunk 1]
```diff
@@ -4018,6 +4017,56 @@ INSERT INTO `synth_recipes` VALUES (3906,1,0,0,114,0,0,0,0,70,0,4099,4241,1394,8
...
+INSERT INTO `synth_recipes` VALUES (3938,1,0,0,0,0,103,0,0,0,0,4096,4238,615,627,2237,4509,5568,5661,0,0,5922,5923,5923,5923,33,33,66,99); -- Walnut Cookie
+INSERT INTO `synth_recipes` VALUES (3939,1,0,0,0,0,104,0,0,0,0,4101,4243,5534,5536,5963,0,0,0,0,0,21492,21493,21493,21493,6,4,8,12); -- Insipid Broth
+INSERT INTO `synth_recipes` VALUES (3940,1,0,0,0,0,104,0,0,0,0,4103,4245,703,2016,4031,4509,0,0,0,0,21494,21495,21495,21495,4,2,4,8); -- Wetlands Broth
+INSERT INTO `synth_recipes` VALUES (3941,1,0,0,0,0,104,0,0,0,0,4096,4238,898,898,4387,4435,4509,5234,9194,0,9197,9197,9197,9197,6,6,10,10); -- Soy Ramen Soup
+INSERT INTO `synth_recipes` VALUES (3942,1,0,0,0,0,104,0,0,0,0,4096,4238,614,617,898,898,4509,5234,9194,0,9198,9198,9198,9198,6,6,8,10); -- Miso Ramen Soup
+INSERT INTO `synth_recipes` VALUES (3943,1,0,0,0,0,104,0,0,0,0,4096,4238,936,1019,4379,4399,4461,4509,5131,0,9199,9199,9199,9199,6,6,6,6); -- Salt Ramen Soup
+INSERT INTO `synth_recipes` VALUES (3944,1,0,0,0,0,105,0,0,0,0,4096,4238,627,2156,4509,5566,0,0,0,0,5926,5927,5927,5927,1,1,1,1); -- Date Tea
```
Comment (Hozu) [inline sql/synth_recipes.sql]: Already exists. https://github.com/DarkstarProject/darkstar/blob/master/sql/synth_recipes.sql#L3658

Comment (R3P0FFXI) [inline sql/synth_recipes.sql]: thanks that's what happens when stuff isn't properly commented

Comment (TeoTwawki) [inline sql/synth_recipes.sql]: thats what happens when you don't control+f in a file that you know is 99% not commented 😛

**Comment by TeoTwawki** [comment]
Rebase "To be finished tomorrow when im at work" out of existence.

**Pull request merged**